pom.xml

```diff
@@ -32,18 +32,11 @@
   <name>WildFly Swarm Booster - ConfigMap Mission</name>
 
   <properties>
-    <version.wildfly.swarm>2017.10.0</version.wildfly.swarm>
-    <version.resteasy>3.0.19.Final</version.resteasy>
-    <version.arquillian.cube>1.9.2</version.arquillian.cube>
-    <!-- Licenses - plugins -->
-    <version.wildfly.swarm.fraction.plugin>61</version.wildfly.swarm.fraction.plugin>
-    <version.xml-maven-plugin>1.0.1</version.xml-maven-plugin>
-    <version.license-maven-plugin>1.13</version.license-maven-plugin>
-    <version.wildfly-licenses-plugin>1.0.1</version.wildfly-licenses-plugin>
-    <!-- Licenses - JBoss repo URL used in custom-settings -->
-    <jboss.repo.url>https://repository.jboss.org/nexus/content/groups/public</jboss.repo.url>
-    <licenses.pom.file>pom.xml</licenses.pom.file>
-    <failOnMissingWebXml>false</failOnMissingWebXml>
+    <version.wildfly.swarm>2018.3.3</version.wildfly.swarm>
+    <version.resteasy>3.0.24.Final</version.resteasy>
+    <version.restassured>3.0.6</version.restassured>
+    <version.awaitility>3.0.0</version.awaitility>
+    <version.openshift.client>3.1.8</version.openshift.client>
```

Comment:
Do we really need to add the dependency on the OpenShift Client?

Reply:
Does not compile otherwise, it is a test dep

Reply:
Aaaaah now I remember! Some parts of the test might be replaced by a calling Arquillian Cube's `OpenShiftAssistant`, but not all of them I guess. Thanks.